readme-vars.yml

```diff
@@ -6,6 +6,7 @@ project_url: "http://flexget.com/"
 project_logo: "https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/flexget-banner.png"
 project_blurb: "[{{ project_name|capitalize }}]({{ project_url }}) is a multipurpose automation tool for all of your media."
 project_lsio_github_repo_url: "https://github.com/linuxserver/docker-{{ project_name }}"
+project_categories: "Media Management"
```

Comment:
I would say it's more a Downloader than Media Mangement as it doesn't really track stuff like the arrs, etc.

Reply:
Done